### ISSUE TITLE
Let users specify the lags the time series pipelines should compute in AutoMLSearch

### ISSUE THREAD

**Issue by freddyaboulton** [alteryx/evalml#2984]
As a user of EvalML for time series problems, I expect to be able to configure a set of lags all time series pipeline should compute. This would let me incorporate my business knowledge into the search.

Came up during the design of #2733.  I think it will be a useful feature but this is nice-to-have for down the road as it's blocked by other issues and it's not part of our beta requirements.

**Comment by dsherry** [comment]
Yep this seems quite helpful.

Don't we already support this in automl search via `pipeline_parameters`? Or would that not work for the time series params?
